README.md

```diff
@@ -9,14 +9,16 @@ A CloudFormation template to deploy YugaByte DB cluster on AWS. It will create a
         $ git clone https://github.com/YugaByte/yugabytedb-cloudformation.git
     ``    
   - change current directory to cloned git repo directory
+  - Make sure you have the latest aws-cli installed
   - Use aws cli to create cloudformation template <br/> 
     `` $ aws cloudformation create-stack 
             --stack-name <your-stack-name> 
-            --template-body yugabyte_cloudformation.yaml 
-            --parameters DBVersion=1.2.8.0,
-                      KeyName=<you-ssh-key-name>
+            --template-body file://.yugabyte_cloudformation.yaml 
```

Comment:
Just confirming that this is indeed `file://.yugabyte_cloudformation.yaml` and not `file://yugabyte_cloudformation.yaml`

Reply:
yes it is